app/assets/stylesheets/login.scss

```diff
@@ -1,101 +1,32 @@
 @import "colors";
 
-@media (min-width: 768px) {
-  .login-page #brand {
-    position: relative;
-    top: -52px;
+.login-pf {
```

Comment:
These styles aren't provided in the patternfly css? Or is this another occurrence of us having to manually copy over styling because of the out of date patternfly-sass project?

Reply:
`patternfly-sass` is up to date in this case.
The styles here represent the differences between our design to patternfly design which is a bit different.
For example, patternfly using a background image, the brand positioning and size are different and some paddings inside the container are different.